src/_templates/footer.html

```diff
@@ -11,7 +11,7 @@
   {% endif %}
 
   <hr/>
-<p class="footerText"><a href="_downloads/ODK-Documentation.pdf" download>Download this documentation as a PDF.</a></p>
+<p class="footerText"><a href="https://docs.getodk.org/_downloads/ODK-Documentation.pdf" download>Download this documentation as a PDF.</a></p>
```

Comment:
This works for the published version! But it will give a false sense of the world with a set of docs hosted somewhere else or locally. Ideally this could dynamically use the base url.